tap-version-14-specification.md

```diff
@@ -0,0 +1,1168 @@
+`Bail out!` line at the root indentation level whenever a Subtest bails
+out.
+
+### Subtest Pragmas
+
+Any Pragmas set in a Subtest affect _only_ the parsing of the Subtest.
+Harnesses _must not_ allow Pragmas set in Subtests to affect the behavior
+of the parser with respect to the parent TAP Document.
+
+For example, given a Harness where a `strict` Pragma will cause it to treat
+any non-TAP as an error:
+
+```tap
+TAP version 14
+pragma -strict
+# Subtest: child test
+    1..1
+    pragma +strict
+    ok 1
+ok 1 - child test
+!!This is not valid TAP content!!
+1..1
+```
+
+In this TAP document, the non-TAP content _must not_ be treated as a test
+failure, because the `strict` Pragma setting at the parent level was false.
+
+### Subtest Parsing/Generating Rules
+
+1. The subtest TAP document is indented 4 spaces.
+2. Subtests _must_ be a valid TAP document, meaning that they cannot be
+   entirely empty.  At minimum, they must include a `1..0` line to indicate
+   that no Test Points are expected.  This implies the following:
+    1. Subtests can be nested within subtests by indenting another 4
+       characters for each level of nesting.
+    2. YAML Diagnostics are indented 2 spaces with respect to the Test
+       Point they are associated with.  So, for example, a YAML Diagnostic
+       block for a Test Point in a nested subtest would be indented 6
+       spaces (4 + 2).  In a subtest nested within another subtest, YAML
+       diagnostics would be indented 10 spaces (4 + 4 + 2).
+3. Subtests _should not_ emit a Version line, if compatibility with TAP13
+   Harnesses is desirable.
+4. The subtest is terminated by a single Test Point line in the parent TAP
+   document, which follows the indented TAP Document.  This is the
+   Correlated Test Point, and reflects the pass/fail status of the Subtest.
+    1. Producers _must_ communicate the intended status of the subtest
+       (pass/fail/todo/skip/etc.) by assigning these semantics to the
+       correlated Test Point.
+    2. Harnesses _should_ treat the entire subtest as either a pass or fail
+       based on the status of the correlated Test Point, but _may_ treat
+       the subtest as a failure if they would consider the nested subtest
+       TAP Document a test failure.
+5. Harnesses _should_ treat otherwise valid TAP that is indented anything
+   other than a multiple of 4 spaces as non-TAP.
+6. Providers _should not_ emit a Version for the Subtest TAP Document.
+7. Providers _should_ emit a Subtest Comment if the name of the subtest is
+   known at the outset.
+8. If Subtest Comment is provided, Harnesses _should_ continue the subtest
+   until a matching test point (same name, or no name in either) at parent
+   level is found, treating any other un-indented lines as non-TAP, and
+   fail the test if a matching test point is not found.
+9. If subtest comment is not provided, Harnesses _must_ treat the next Test
```

Comment:
```suggestion
9. If Subtest Comment is not provided, Harnesses _must_ treat the next Test
```
nit: adding a bit of internal consistency to this numbered list.

Reply:
Yes, "Subtest Comment" is a Defined Term, so it should be capitalized whenever it's used in this manner. A specification is the place to indulge all your worse nit-picking impulses :+1: